### PR TITLE
setCustomProperties method hidden

### PR DESCRIPTION
**Pull request by bmourat**
MSMobileCenter setCustomProperties method moved to MSMobileCenterPrivate until backend is ready.

### CODE REVIEW

**Comment by jaelim-ms** [comment]
Build is broken due to project configuration. Please merge develop.

**Comment by bmourat** [comment]
This PR is for base/push branch

**Comment by jaelim-ms** [comment]
OK then you have to merge base/push branch. I'm still working on it.

File: MobileCenter/MobileCenter/Internals/MSMobileCenterPrivate.h [hunk 1]
```diff
@@ -5,4 +5,11 @@
...
-@end
\ No newline at end of file
+// TODO: Move to MSMobileCenter.h when backend is ready.
+/**
+ * Set the custom properties.
+ *
+ * @param customProperties Custom properties object.
+ */
++ (void)setCustomProperties:(MSCustomProperties *)customProperties;
+@end
```
Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/MSMobileCenterPrivate.h]: Add an empty line before `@end`

Comment (jaelim-ms) [inline MobileCenter/MobileCenter/Internals/MSMobileCenterPrivate.h]: TODO can be placed right after doc, right before method signature.

**Pull request merged**